tvm/op/stack/blkpush.go

```diff
@@ -0,0 +1,53 @@
+package stack
+
+import (
+	"fmt"
+
+	"github.com/xssnick/tonutils-go/tvm/cell"
+	"github.com/xssnick/tonutils-go/tvm/op/helpers"
+	"github.com/xssnick/tonutils-go/tvm/vm"
+	"github.com/xssnick/tonutils-go/tvm/vmerr"
+)
+
+func init() {
+	vm.List = append(vm.List, func() vm.OP { return BLKPUSH(0, 0) })
+}
+
+func BLKPUSH(i, j uint8) (op *helpers.AdvancedOP) {
+	op = &helpers.AdvancedOP{
+		Action: func(state *vm.State) error {
+			if state.Stack.Len() < int(j) {
+				return vmerr.ErrStackUnderflow
+			}
+			for x := i; x != 0; x-- {
+				if err := state.Stack.PushAny(state.Stack.Get(j)); err != nil {
+					return err
+				}
+			}
+			return nil
+		},
+		NameSerializer: func() string {
+			return fmt.Sprintf("%d, %d BLKPUSH", i, j)
+		},
+		Prefix: cell.BeginCell().MustStoreSlice([]byte{0x5F}, 8).EndCell(),
+		SerializeSuffix: func() *cell.Builder {
+			return cell.BeginCell().MustStoreUInt(uint64(i), 4).MustStoreUInt(uint64(j), 4)
+		},
+		DeserializeSuffix: func(code *cell.Slice) error {
+			val, err := code.LoadUInt(4)
+			if err != nil {
+				return err
+			}
+			i = uint8(val)
+
+			val, err = code.LoadUInt(4)
+			if err != nil {
+				return err
+			}
+			j = uint8(val)
```

Comment:
better to set i and j both when successful execution is guaranted, to not have partial result in case of second loaduint fail.

(same in other places)